Libs/MRML/Core/vtkMRMLAbstractViewNode.cxx

```diff
@@ -333,29 +328,18 @@ bool vtkMRMLAbstractViewNode::SetInteractionNode(vtkMRMLNode* node)
   return this->SetInteractionNodeID(node ? node->GetID() : nullptr);
 }
 
-int vtkMRMLAbstractViewNode::IsMappedInLayout()
-{
-  if (!this->GetAttribute("MappedInLayout"))
-  {
-    return 0;
-  }
-  return strcmp(this->GetAttribute("MappedInLayout"), "1") == 0;
-}
-
 //------------------------------------------------------------------------------
-void vtkMRMLAbstractViewNode::SetMappedInLayout(int value)
+int vtkMRMLAbstractViewNode::IsMappedInLayout()
```

Comment:
Doing some further testing, I think the mapped in layout change is also required for vtkMRMLAbstractViewNode sequences. It would be great if this could also be split.

Reply:
ok I will add it to https://github.com/Slicer/Slicer/pull/8370

Reply:
done in https://github.com/Slicer/Slicer/pull/8370/commits/5d8691e7429a4593c6b99a0394f151747ab7c09f